exporter/fileexporter/file_exporter_test.go

```diff
@@ -673,3 +674,86 @@ func TestFlushing(t *testing.T) {
 	assert.EqualValues(t, b, bbuf.Bytes())
 	assert.NoError(t, fe.Shutdown(ctx))
 }
+
+func TestAppend(t *testing.T) {
```

Comment:
This test is a bit too low level to my taste, using `fileWriter` directly. I think it's because it was based on the `TestFlushing` test. Perhaps better to write this test more like the `TestFileTracesExporter` and similar tests are written:

1. create `newFileExporter()`
2. generate traces
3. write telemetry with `consumeTraces`
4. check that traces written (check number of lines with JSON or sth)
5. create a new instance of `newFileExporter`
6. `consumeTraces`
7. check the cumulative number of lines in file